src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -154,6 +155,35 @@ namespace OpenLoco
         2,
     };
 
+    struct ThoughtMinMaxVehicles
+    {
+        uint8_t min;
+        uint8_t max;
+    };
+    // 0x004FE784 & 0x004FE785
+    static constexpr std::array<ThoughtMinMaxVehicles, kAiThoughtTypeCount> kThoughtTypeMinMaxNumVehicles = {
```

Comment:
I think this would benefit from using [std::to_array](https://en.cppreference.com/w/cpp/container/array/to_array) so you don't need to repeat the type on every line.